### ISSUE TITLE
securedrop_init.py: uses System V instead of systemd

### ISSUE THREAD

**Issue by ageis** [freedomofpress/securedrop#1281]
In tails_files, we have a setuid Python script that reloads Tor and writes the SecureDrop-specific configuration. Tails 2.x, based on Debian 8/jessie, now uses systemd, where the convention for reloading services is `systemctl`.

This should eventually be updated.
```
    subprocess.call(['/usr/sbin/service', 'tor', 'reload']) 
```

**Comment by garrettr** [comment]
I'm surprised this still works at all. A guess: Debian adds some symlinks for backwards compatibility?

**Comment by garrettr** [comment]
Since we're going to be asking everyone to re-run `tails_files/install.sh` soon (to upgrade to 2.x), might as well take care of this now.

**Comment by ageis** [comment]
Tor itself still ships with init scripts, so service reload works. On my system anyway, there is not a full Tor systemd service unit, although the code for that is [out there](https://github.com/torproject/tor/blob/b48f8a8114575793046a8e9d3544d6b2d0a17f82/contrib/dist/tor.service.in) and seems to be in various states of implementation across distros. The systemd target relies on the init script (backwards compatbility, yup).

**Comment by garrettr** [comment]
https://github.com/freedomofpress/securedrop/issues/1281#issuecomment-214506419 turned out to be incorrect, see discussion in https://github.com/freedomofpress/securedrop/pull/1285#issuecomment-217241928. This issue was fixed in #1285.
